Formula/ptpython.rb

```diff
@@ -49,12 +50,12 @@ class Ptpython < Formula
   end
 
   def install
-    venv = virtualenv_create(libexec, Formula["python@3.9"].opt_bin/"python3")
+    venv = virtualenv_create(libexec, Formula["python@3.10"].opt_bin/"python3")
     venv.pip_install resources
     venv.pip_install buildpath
 
     # Make the Homebrew site-packages available in the interpreter environment
```

Comment:
This workaround isn't actually needed anymore, since we pass `--system-site-packages` when we create the venv.